test/uk/gov/hmrc/basisperiodreformapi/controllers/BasisPeriodReformControllerSpec.scala

```diff
@@ -66,4 +66,39 @@ class BasisPeriodReformControllerSpec extends HmrcSpec with ResetMocksAfterEachT
       GetPartnershipDetails.verifyNotCalled()
     }
   }
+  "GET /views/iv_overlap_relief_sole_trader" should {
```

Comment:
Pedantic alert: a blank line before this would be nice, and before `"return 200 when..."` a few lines down.